staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -587,19 +587,32 @@ func (r *Request) tryThrottle(ctx context.Context) error {
 	err := r.rateLimiter.Wait(ctx)
 
 	latency := time.Since(now)
+
+	var message string
+	switch {
+	case len(retryInfo) > 0:
+		message = fmt.Sprintf("Waited for %v due to client-side throttling, request: %s:%s - %s", latency, r.verb, r.URL().String(), retryInfo)
```

Comment:
In this case we can get a message that says both "due to client-side throttling" and "due to server-side throttling" --- right?  I think that is going to be confusing.

Reply:
I dropped `client-side throttling` - so the message looks like:
`Waited for 1.050387818s, request: POST:https://request URI- retries: 3, retry-after: 1s - retry-reason: due to server-side throttling, FlowSchema UID: ""`